rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/jboss/GenerateJBossWebDescriptorRuleProvider.java

```diff
@@ -37,53 +36,46 @@
     public static final String JBOSS_WEB_TEMPLATE = "/reports/templates/jboss/jboss-web.ftl";
 
-    public GenerateJBossWebDescriptorRuleProvider()
-    {
-        super(MetadataBuilder.forProvider(GenerateJBossWebDescriptorRuleProvider.class, "Generate jboss-web.xml")
-                    .setPhase(MigrationRulesPhase.class));
-    }
-
     @Override
     public Configuration getConfiguration(GraphContext context)
     {
         return ConfigurationBuilder.begin()
-                    .addRule()
-                    .when(Query.fromType(WebXmlModel.class))
-                    .perform(new GraphOperation()
-                    {
-                        @Override
-                        public void perform(GraphRewrite event, EvaluationContext context)
-                        {
-                            // configuration of current execution
-                            WindupConfigurationModel configurationModel = WindupConfigurationService.getConfigurationModel(event.getGraphContext());
-
-                            for (FileModel inputPath : configurationModel.getInputPaths())
-                            {
-                                ProjectModel projectModel = inputPath.getProjectModel();
-                                transformWebXml(context, event.getGraphContext(), projectModel);
-                            }
-                        }
-
-                        @Override
-                        public String toString()
-                        {
-                            return "Generate jboss-web.xml";
-                        }
-                    });
+        .addRule()
+        .when(Query.fromType(WebXmlModel.class))
+        .perform(new GraphOperation()
+        {
+            @Override
+            public void perform(GraphRewrite event, EvaluationContext context)
+            {
+                // configuration of current execution
+                WindupConfigurationModel configurationModel = WindupConfigurationService.getConfigurationModel(event.getGraphContext());
+
+                for (FileModel inputPath : configurationModel.getInputPaths())
+                {
+                    ProjectModel projectModel = inputPath.getProjectModel();
+                    transformWebXml(context, event.getGraphContext(), projectModel);
+                }
+            }
 
+            @Override
+            public String toString()
+            {
+                return "Generate jboss-web.xml";
+            }
+        });
     }
 
     private void transformWebXml(EvaluationContext evaluationContext, GraphContext context, ProjectModel projectModel)
     {
-        GraphService<WebXmlModel> webDescriptors = new GraphService<>(context, WebXmlModel.class);
-        ClassificationService classificationService = new ClassificationService(context);
+        //GraphService<WebXmlModel> webDescriptors = new GraphService<>(context, WebXmlModel.class);
```

Comment:
Can you remove the commented code, please?

Not a big deal, but it looks like this needs a rebase anyway.